### DISCUSSION TITLE
Cannot preview markdown

### DISCUSSION THREAD

**Discussion by avi12** [community/community#37077] [Codespaces]
When attempting to load [README.md](https://github.dev/guest271314/captureSystemAudio), I get
```
Error loading webview: Error: Could not register service workers: NotSupportedError: Failed to register a ServiceWorker for scope ('https://1ukoupo0eu36gj8ue4ds8bui08qp3fnslcngdqmqcpue4p9hgg7l.vscode-cdn.net/stable/f69482e92e41b84fb15688f3c0d0a1eca1d5bce3/out/vs/workbench/contrib/webview/browser/pre/') with script ('https://1ukoupo0eu36gj8ue4ds8bui08qp3fnslcngdqmqcpue4p9hgg7l.vscode-cdn.net/stable/f69482e92e41b84fb15688f3c0d0a1eca1d5bce3/out/vs/workbench/contrib/webview/browser/pre/service-worker.js?v=4&vscode-resource-base-authority=vscode-resource.vscode-cdn.net&remoteAuthority='): The user denied permission to use Service Worker..
```

**Comment by cmbrose** [comment]
Hi @avi12 - thanks for reaching out! Could you let us know which browser you are using? Some codespaces functionality requires a service worker to run, but we have seen that some browsers like Firefox or Brave might be overly restrictive and cause an issue. Could you also try again after disabling browser extensions as those can sometimes also interfere. Thanks!

**Reply by avi12** [reply]
Brave

**Reply by cmbrose** [reply]
Thanks - I can repro in Brave. If you disable the Shield for the site you will be able to see the render, or more specifically if you enable cookies in the Shield advanced options it will work. It's not cookies specifically that are breaking things, but it looks like Brave groups the service worker enablement with the cookie enablement.

**Reply by avi12** [reply]
Just found out that the preview doesn't fully support dark mode
![image](https://user-images.githubusercontent.com/6422804/197634321-1a581d7b-afa8-4c7f-8bf7-dc1e3d6c223d.png)

**Reply by cmbrose** [reply]
@avi12 dark mode behavior will need to be handled on the vscode side. FWIW I also use dark themes and haven't experienced this, so perhaps it is something wrong in the specific theme that you use. In any case, if you believe that is a bug we will need to file something in https://github.com/microsoft/vscode/issues, but that is a different topic than this issue.

**Comment by ossanna16** [comment]
Hi there @avi12 and welcome to our community! Thank you for asking a great question 🙂
